packages/react/src/drawer/Drawer.css.ts

```diff
@@ -0,0 +1,71 @@
+import { theme } from "../styles";
+import { type RecipeVariants, recipe, style } from "../vanilla-extract";
+
+export const content = recipe({
+  base: [
+    {
+      bg: "surface",
+      display: "flex",
+      flexDirection: "column",
+      shadow: "md",
+      z: "popover",
+    },
+    style({
+      position: "fixed",
+    }),
+  ],
+  variants: {
+    position: {
+      bottom: style({
+        borderTop: `1px solid ${theme.colors["border.secondary"]}`,
```

Comment:
let's use `borderT` sprinkle prop for this and the other right/left borders in the other variants